npm/test/basics.js

```diff
@@ -73,6 +73,36 @@ namespace Test {
   assert(result.result === "Zero");
 });
 
+test("one syntax error", async () => {
+  const compiler = getCompiler();
+
+  const diags = await compiler.checkCode("namespace Foo []");
```

Comment:
This may have already been discussed, but it looks like `checkCode` is deprecated and the messages says we should use the langauge service instead. Is that something we can do yet, or will we need to use this until we have langauge service integration farther along?